bootstrap/config/kfctl_aws.yaml

```diff
@@ -0,0 +1,164 @@
+apiVersion: kfdef.apps.kubeflow.org/v1alpha1
+kind: KfDef
+metadata:
+  name: kubeflow-aws
+  namespace: kubeflow
+spec:
+  platform: aws
+  componentParams:
```

Comment:
The fields packages, components, componentParams are the deprecated fields that were there for ksonnet.

If you want you can still use these fields but since its a new config I might suggest using the new field

Spec.applications

Each application is just a pointer to a kustomize package.

kfctl will automatically update kfdef.spec.applications based on packages, components,componentParams

so if you run kfctl and then look at app.yaml you can copy the value of KfDef.Spec.Applications from there.

Reply:
Sure. I will make some clean ups as you suggest. 
1. Define a plugin to avoid bring in more options
2. Use Applications instead.

Reply:
Thanks